ui/src/app/pages/analysis/notebook-resource-card.tsx

```diff
@@ -146,6 +160,8 @@ export const NotebookResourceCard = fp.flow(
   }
 
   copyNotebook(copyRequest: CopyRequest) {
+    AnalyticsTracker.Notebooks.Copy();
+
```

Comment:
nit: extra line

Reply:
I was hoping to keep this line to space out different blocks of logic.